Projects/Payment/Sources/Models/PaymentStatusData.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import hGraphQL
+
+public struct PaymentStatusData: Codable, Equatable {
+    public var status: PayinMethodStatus = .active
+    public var failedCharges: Int?  //TODO: FIX this
```

Comment:
What to fix? Is it something that should be fixed from backend?

Reply:
We should handle it somehow but will release as it is now. WIl update with new payments views